src/main/kotlin/uk/gov/dluhc/printapi/service/SftpService.kt

```diff
@@ -1,6 +1,8 @@
 package uk.gov.dluhc.printapi.service
 
+import com.fasterxml.jackson.databind.ObjectMapper
 import com.jcraft.jsch.ChannelSftp
+import com.nimbusds.jose.util.IOUtils
```

Comment:
Is ObjectMapper still used in this class?

Is there a better IOUtils to use here? One from apache commons or springframework perhaps?
This one from nimbusds.jose is part of Spring Security, and specifically the JWT support stuff I think ?

Reply:
Thanks, I hadn't spotted that rogue import.